Formula/dafny.rb

```diff
@@ -21,10 +22,13 @@ class Dafny < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "25d87dc6b3a2041668a3e4100d3e44c61bee2a772a8405fdc8fc63d1ac35a200"
   end
 
-  depends_on "gradle" => :build
```

Comment:
`./gradlew` is run during `make exe` as part of C# project https://github.com/dafny-lang/dafny/blob/v3.9.1/Source/DafnyRuntime/DafnyRuntime.csproj#L47
```csproj
    <Exec WorkingDirectory="DafnyRuntimeJava" Condition="!$([MSBuild]::IsOSPlatform('Windows'))" Command="./gradlew -q build" />
```